lib/Remotery.h

```diff
@@ -661,6 +691,23 @@ RMT_API void _rmt_BeginMetalSample(rmtPStr name, rmtU32* hash_cache);
 RMT_API void _rmt_EndMetalSample(void);
 #endif
 
+// Iterator
+RMT_API rmtSampleIterator   _rmt_IterateChildren(rmtSample* sample);
+RMT_API rmtBool             _rmt_IterateNext(rmtSampleIterator* iter);
+
+// SampleTree accessors
+RMT_API const char*         _rmt_SampleTreeGetThreadName(rmtMsgSampleTree* sample_tree);
+RMT_API rmtSample*          _rmt_SampleTreeGetRootSample(rmtMsgSampleTree* sample_tree);
+
+// Sample accessors
+RMT_API const char*         _rmt_SampleGetName(Remotery* rmt, rmtSample* sample);
+RMT_API rmtU32              _rmt_SampleGetNameHash(rmtSample* sample);
+RMT_API rmtU32              _rmt_SampleGetCallCount(rmtSample* sample);
+RMT_API rmtU64              _rmt_SampleGetTime(rmtSample* sample);
+RMT_API rmtU64              _rmt_SampleGetSelfTime(rmtSample* sample);
+RMT_API void                _rmt_SampleGetColour(rmtSample* sample, rmtU8* r, rmtU8* g, rmtU8* b);
+RMT_API rmtSampleType       _rmt_SampleGetType(rmtSample* sample);
```

Comment:
Currently, they all are "private", with the "_" prefix.

I guess they should also be wrapped with the "RMT_OPTIONAL" define.
Should it also be contained withing a new tag, e.g. `RMT_ENABLED_ITERATORS` ? Might be a bit overkill?

Reply:
Don't worry about adding a `RMT_ENABLED_ITERATORS` - just `RMT_ENABLED` will do.